src/PrestaShopBundle/Resources/views/Admin/Multistore/header.html.twig

```diff
@@ -24,7 +24,15 @@
  *#}
 {% block multistore_header %}
   {% if isMultistoreUsed and app.request is not null %}
-    <div class="header-multishop{% if isAllShopContext %} header-multishop-allshops{% elseif currentContext.color is empty %} header-multishop-default{% endif %}{% if isTitleDark is same as (true) %} header-multishop-dark{% else %} header-multishop-bright{% endif %}">
+    <div
+            class="header-multishop{% if isAllShopContext %} header-multishop-allshops{% elseif currentContext.color is empty %} header-multishop-default{% endif %}{% if isTitleDark is same as (true) %} header-multishop-dark{% else %} header-multishop-bright{% endif %}"
+            data-translations="{{ {
```

Comment:
@sowbiba 

Wouldn't it be better to follow the same style for translations as in other views? Example:

```
{% set js_translatable = {
  "Customer search - group default": 'default'|trans({}, 'Admin.Orderscustomers.Feature'),
  "Customer search - group label single": 'Group'|trans({}, 'Admin.Orderscustomers.Feature'),
  "Customer search - group label multiple": 'Groups'|trans({}, 'Admin.Orderscustomers.Feature'),
}|merge(js_translatable)
%}
```

`window.translate_javascripts['Customer search - group label multiple']`